src/utils.cpp

```diff
@@ -514,6 +514,18 @@ int modify_target_rbuf_raw(struct char_data *ch, char *rbuf, size_t rbuf_len, in
     buf_mod(rbuf, rbuf_len, "Sunlight", 1);
     WRITEOUT_MSG("Sunlight Allergy", 1);
   }
+  if (IS_GHOUL(ch) && ((time_info.hours > 6) && (time_info.hours < 19)) && OUTSIDE(ch) && weather_info.sky < SKY_RAINING)
```

Comment:
Is there a difference between this check and the RACE_NIGHTONE above? If not, recommend removing the duplicated code and changing the check for the previous one instead.

`if ((GET_RACE(ch) == RACE_NIGHTONE || IS_GHOUL(ch)) && ((time_info.hours...`

Reply:
Nope, I just separated em it while testing, merged